bin/utils/TablePrinter.js

```diff
@@ -1,79 +1,99 @@
 'use strict';
 
-const Table = require('cli-table2');
-
 const MIN_CELL_WIDTH = 15;
 
 class TablePrinter {
   constructor(opts) {
     this.opts = opts;
     this._hasWritten = false;
-    this.colWidths;
   }
 
   push(csv) {
     const lines = csv.split(this.opts.eol);
 
-    const chars = {
-      'bottom': '',
-      'bottom-mid': '',
-      'bottom-left': '',
-      'bottom-right': ''
-    };
+    if (!lines.length) return;
 
     if (!this._hasWritten) {
-      this.colWidths = this.getColumnWidths(lines[0]);
-      if (this.opts.header) {
-        const head = lines.shift().split(this.opts.delimiter);    
-        const table = new Table({ head, colWidths: this.colWidths, chars });
-        this.print(table, []);
-        this._hasWritten = true;
-      }
-    } else {
-      chars['top-left'] = '├';
-      chars['top-mid'] = '┼';
-      chars['top-right'] = '┤';
+      this.setColumnWidths(lines[0]);
     }
 
-    if (!lines.length) return;
-
-    const table = new Table({ colWidths: this.colWidths, chars });
-    this.print(table, lines);
+    const top = this._hasWritten ? this.middleLine : this.topLine;
+    this.print(top, lines);
     this._hasWritten = true;
   }
 
   end(csv) {
     const lines = csv.split(this.opts.eol);
-    const chars = { 'top-left': '├' , 'top-mid': '┼', 'top-right': '┤' };
-    const table = new Table({ colWidths: this.colWidths, chars });
-    this.print(table, lines);
+    this.print(this.middleLine, lines, this.bottomLine);
   }
 
-  printCSV(csv) {
+  printCSV(csv) { // SYNC
```

Comment:
Should this comment be expanded?

Reply:
Yeah!
I just removed it. It was a pretty unnecessary comment imo 🙂